src/extra/stream/client.test.ts

```diff
@@ -1,22 +1,94 @@
 // @vitest-environment happy-dom
 
-import { describe, expect, test } from 'vitest';
+import { vi, beforeEach, afterEach, describe, expect, test } from 'vitest';
 import { RobotClient } from '../../robot';
 import { events } from '../../events';
+import { StreamServiceClient } from '../../gen/proto/stream/v1/stream_pb_service';
 import { StreamClient } from './client';
 
+let robotClient: RobotClient;
+let streamClient: StreamClient;
+
 describe('StreamClient', () => {
+  beforeEach(() => {
+    vi.useFakeTimers();
+
+    vi.mock('./robot/client');
```

Comment:
What's this trying to mock out and why?

Reply:
The stream client requires a robot client to initialize